lib/measures/default_feature_reports/measure.rb

```diff
@@ -762,12 +762,14 @@ def run(runner, user_arguments)
                           'm3'
                       end
         end
+        
         # loop through each value and apply unit conversion
         os_vec = values[key_cnt]
         if !timeseries_name.include? 'Zone Thermal Comfort'
           for i in 0..os_vec.length - 1
-            unless new_unit == old_unit || !ts.is_initialized
-              os_vec[i] = OpenStudio.convert(os_vec[i], old_unit, new_unit).get
+
+          unless new_unit == old_unit || old_unit.nil?  || new_unit.nil? || !ts.is_initialized || os_vec[i].nil?
```

Comment:
I don't think you have to check os_vec[i].nil? since you are in the for loop already. otherwise it looks good!